sovrin_node/persistence/attribute_store.py

```diff
@@ -0,0 +1,25 @@
+from plenum.persistence.kv_store_leveldb import KVStoreLeveldb
+
+
+class AttributeStore:
+    """
+    Stores attributes as key value pair where the key is hash of the
+    attribute as stored in ledger and value is the actual value if the attribute
+    """
+
+    def __init__(self, dbPath):
+        self.dbPath = dbPath
+        self.db = KVStoreLeveldb(dbPath)
```

Comment:
If we need to change a KVStore type, we need to change implementation. It's better to inject `db` into constructor (in addition to `dbpah`), so that using another DB (for example RocksDB) can be done easily using existing `AttributeStore` class.